library/libraries.js

```diff
@@ -1797,6 +1797,27 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             if (win.__GUESS__ && win.__GUESS__.guess) {
                 return { version: UNKNOWN_VERSION };
             }
+            return false;
+        }
+    },
+    'October CMS': {
+        id: 'octobercms',
+        icon: 'octobercms',
+        url: 'https://octobercms.com/',
+        npm: null,
+        test: function (win) {
+            const generatorMeta1 = document.querySelector('meta[name="generator"][content^="OctoberCMS"]');
+            const generatorMeta2 = document.querySelector('meta[name="generator"][content^="October CMS"]');
+            
+            // October CMS resource patterns / paths - search in link, style or script tags
+            const resourcesOctober = /\/modules\/system\/assets\/(css|js)\/framework(\.extras|\.combined)?(-min)?/;
```

Comment:
Seems like a good thing to search for. https://www.google.com/search?sxsrf=ALeKk012q_oauhBTpZmw_jnrrK-Ux-mW8g%3A1606180466075&ei=cl68X4-VBMKUsgXM2KC4Bw&q=%22modules%2Fsystems%2Fassets%2Fjs%22&oq=%22modules%2Fsystems%2Fassets%2Fjs%22&gs_lcp=CgZwc3ktYWIQAzIFCCEQqwIyBQghEKsCMgUIIRCrAjoRCAAQsAMQigMQtwMQ5QIQiwNQpR1YiiBglSJoAXAAeACAAW2IAcMBkgEDMS4xmAEAoAEBqgEHZ3dzLXdpesgBCrgBAsABAQ&sclient=psy-ab&ved=0ahUKEwiPloDBgJrtAhVCiqwKHUwsCHcQ4dUDCA0&uact=5

Reply:
Yah it's pretty good see screenshot:

![image](https://user-images.githubusercontent.com/57409060/100102765-f4450e00-2e5b-11eb-9674-73727015b501.png)